packages/schema/exported-schema.json

```diff
@@ -695,6 +695,30 @@
               "value": "**yes** (with exceptions, see description)"
             }
           }
+        },
+        "lock": {
+          "description": "**INTERNAL USE ONLY**. Zapier uses this config for internal operation locking.",
+          "type": "object",
+          "required": ["key"],
+          "properties": {
+            "key": {
+              "description": "The key to use for locking. This should be unique to the operation.",
+              "type": "string",
+              "minLength": 1
+            },
+            "scope": {
+              "description": "The level at which an app's access is restricted to. \n              'user' - Locks based on user ids. \n              'auth' - Locks based on unique auth ids\n              'account' - Locks for all users under a single account. \n              You may also combine scopes.",
```

Comment:
Although I suggested removing `app` from the scope list, I think we should be transparent about what it means when we default to it when no scope is specified. And if referring to it in the description w/o being in the list could be confusing, perhaps we add it also.